File: Two Sum/Two_Sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        length = len(nums)
        
        for i in range(length):
            needed = target - nums[i]
            for j in range(i+1, length):
                if(nums[j] == needed):
                    return i, j
        

File: Two Sum/Two_Sum.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        vector<int> indexs(2);
	for (int i = 0; i < nums.size(); i++) {
		for (int j = i+1; j < nums.size(); j++)
		if (nums[i] + nums[j] == target) {
			indexs[0] = j  ;
			indexs[1] = i ;
			return indexs;
		}
	}
	return indexs;
    }
};